app/domain/authentication/authn_jwt/vendor_configurations/configuration_jwt_generic_vendor.rb

```diff
@@ -129,8 +130,9 @@ def extract_resource_restrictions
         end
 
         def constraints
-          @constraints ||= Authentication::Constraints::MultipleConstraint.new(
-            Authentication::Constraints::NotEmptyConstraint.new
+          @constraints ||= @create_constraints_from_policy.call(
```

Comment:
i think you should replace the `NonPermittedConstraint` dependency and create it here with its input: `MANDATORY_CLAIMS_DENY_LIST `

Instead of pass the input `MANDATORY_CLAIMS_DENY_LIST ` 


Or if this input is CONST why not create it in the `CreateConstrainsFromPolicy` command class in the `dependencies` section ?

Reply:
Done